auction-server/src/auction/service/simulator.rs

```diff
@@ -168,6 +168,8 @@ impl Simulator {
         Ok(result.value)
     }
 
+    /// Fetches multiple accounts from the RPC in chunks
+    /// There is no guarantee that all the accounts will be fetched with the same slot
```

Comment:
i think this is still weird, but i don't have a great solution for it atm. maybe highlight this as a TODO to think about downstream simulation issues

Reply:
if the max rpc account limit is 100, should be fine though? max of 64 accounts per transaction, where did you get the max limit of 100 from?

Reply:
Based on the docs:
https://solana.com/docs/rpc/http/getmultipleaccounts

We try to fetch all the accounts needed for multiple transactions, so we might need chunking.